content/blog/2025-05-08-go-client-GA.md

```diff
@@ -0,0 +1,519 @@
+// Execute transaction with context
+results, err := client.Exec(context.Background(), *tx, false)
+```
+
+### Optimistic Concurrency Support
+
+Valkey's optimistic locking mechanism is supported through:
+
+- Implementation of `Watch()` that monitors keys for changes,
+- Implementation of `Unwatch()` that releases monitoring,
+- Proper handling of transaction failure when watched keys change,
+- Passing watch state through to the Valkey server during execution.
+
+This enables patterns like:
+
+```go
+client.Watch(context.Background(), []string{"key1"})
+tx := pipeline.NewStandaloneBatch(true).
+    Get("key1").
+    Set("key1", "new-value")
+
+// Will fail if key1 changed between Watch and Exec
+raiseOnError := true
+results, err := client.Exec(context.Background(), *tx, raiseOnError)
+```
+
+## Join the Journey
+
+We're actively developing and enhancing the Go client, and we'd love your feedback and contributions. Try it out in your projects, share your experiences, and help us make it even better!
+You can join our development journey by:
+
+- Submitting issues or feature requests on our [GitHub Issues page](https://github.com/valkey-io/valkey-glide/issues)
+- Joining discussions in our [GitHub Discussions forum](https://github.com/valkey-io/valkey-glide/discussions)
+
+## Looking Forward
+
+As we move forward, we'll continue enhancing performance, and adding even more features to the Valkey GLIDE Go client to continue building on the great foundation provided by Valkey GLIDE 2.0.
+
+Checkout our [Valkey GLIDE go client](https://github.com/valkey-io/valkey-glide/tree/main/go) for the source code.
+For more details on how to get started with Valkey GLIDE Go client, please refer to the [README](https://github.com/valkey-io/valkey-glide/blob/main/go/README.md) for instructions on running the Standalone and Cluster examples.
+
+For a complete reference of all available commands and their parameters, explore the [Go API documentation on pkg.go.dev](https://pkg.go.dev/github.com/valkey-io/valkey-glide/go/v2), which provides detailed information on method signatures, parameters, and return types.
+
+## Contributors
+
+A huge thank you to all the contributors and community members who have made this possible - your dedication and expertise have created something truly special for the Go community.
+
+[Bar Shaul](https://github.com/barshaul) (Amazon Web Services)
+
+[Shoham Elias](https://github.com/shohamazon) (Amazon Web Services)
+
+[Adar Ovadia](https://github.com/adarovadya) (Amazon Web Services)
+
+[Meital Krasikow](https://github.com/meitalkra) (Amazon Web Services)
+
+[Asaf Porat Stoler](https://github.com/asafpamzn) (Amazon Web Services)
+
+[Janhavi Gupta](https://github.com/janhavigupta007) (Google Cloud Platform)
+
+[Niharika Bhavaraju](https://github.com/niharikabhavaraju) (Google Cloud Platform)
+
+[Edric Cuartero](https://github.com/EdricCua) (Google Cloud Platform)
+
+[Omkar Mestry](https://github.com/omangesg) (Google Cloud Platform)
```

Comment:
Can you please update this to https://github.com/omanges

Reply:
Done